process/sync/storageBootstrap/baseStorageBootstrapper.go

```diff
@@ -206,6 +212,44 @@ func (st *storageBootstrapper) loadBlocks() error {
 	return nil
 }
 
+func (st *storageBootstrapper) addReferencedMetablocksInTracker(header bootstrapStorage.BootstrapHeaderInfo) error {
+	if header.ShardId == common.MetachainShardId {
+		return nil
+	}
+
+	hdr, err := st.bootstrapper.getHeader(header.Hash)
+	if err != nil {
+		return err
+	}
+
+	shardHeader, ok := hdr.(data.ShardHeaderHandler)
```

Comment:
I think we should replace L225-L228 with:

```
	prevHdr, err := st.bootstrapper.getHeader(hdr.GetPrevHash())
	if err != nil {
		return err
	}
	
	shardHeader, ok := prevHdr.(data.ShardHeaderHandler)
	if !ok {
		return process.ErrWrongTypeAssertion
	}

```

As we need to add the metablocks and metablocks hashes contained in the previous header from the one we will started. All the info regarding the headerInfo.LastHeader is already added in the last call of applyBootInfos

Reply:
forgot about previous header :man_facepalming: